src/main/java/edu/ucla/library/prl/harvester/JobResult.java

```diff
@@ -49,17 +61,29 @@ public JobResult(final OffsetDateTime aStartTime, final int aRecordCount) {
      * Instantiates a job result from its JSON representation.
      * <p>
      * <b>This constructor is meant to be used only by generated service proxy code!</b>
```

Comment:
They are, although hypothetically there could be a MessageConsumer running outside of a service that receives serialized-as-JSON JobResults on `HarvestJobSchedulerService.JOB_RESULT_ADDRESS`. I think I'd rather remove the dictum, assuming it will be obvious to the user which constructor to use in a given context (and that there's no worry since they're thoroughly tested).